content/configuration/webhooks.md

```diff
@@ -15,6 +15,8 @@ You can find your app ID in the browser URL after `app/` when you open the app o
 
 All received webhooks are visible in the Codemagic UI when navigating to your application and selecting the **Webhooks** tab.
 
+After configuring **Webhooks**, automatic build triggering can be set up as explained [here](https://docs.codemagic.io/yaml/yaml-getting-started/#triggering)
+
```

Comment:
I think that it needs to be two links. 
- One for how to set up triggers in Workflow Editor
- One for how to set it up in codemagic.yaml